electrode_models/air_electrode.py

```diff
@@ -413,6 +414,9 @@ def residual(self, t, SV, SVdot, sep, counter, params):
             + sdot_elyte_host * A_surf_ratio)
             * self.dyInv)/eps_elyte
 
+        eps_elyte = 1. - eps_product - self.eps_host[0]
+        self.elyte_microstructure = eps_elyte**1.5
```

Comment:
I don't know that these lines do anything...

Reply:
resets node to zero, because it is passed to separator as the final node.